Modules/_ctypes/_ctypes.c

```diff
@@ -3913,9 +3928,11 @@ _validate_paramflags(ctypes_state *st, PyTypeObject *type, PyObject *paramflags)
                         "abstract class");
         return 0;
     }
-    argtypes = info->argtypes;
+    if (argtypes == NULL || argtypes == Py_None) {
+        argtypes = info->argtypes;
+    }
 
-    if (paramflags == NULL || info->argtypes == NULL)
+    if (paramflags == NULL || argtypes == NULL)
         return 1;
```

Comment:
Please add braces when updating an `if` statement.
```suggestion
    if (paramflags == NULL || argtypes == NULL) {
        return 1;
    }
```